src/components/item/form/link/LinkForm.tsx

```diff
@@ -213,7 +213,9 @@ const LinkForm = ({
                   <LinkCard
                     title={linkData?.title || ''}
                     url={linkContent}
-                    thumbnail={linkData?.thumbnails[0]}
+                    thumbnail={
+                      linkData?.thumbnails?.[0] ?? linkData?.icons?.[0]
```

Comment:
Should we handle empty string here ?

```suggestion
                      linkData?.thumbnails?.[0] || linkData?.icons?.[0]
```